core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -122,9 +122,14 @@ export class ServiceDetailComponent implements OnInit {
         endpoints: service.endpoints,
         deployment_targets :  service.deployment_targets[service.type].S || service.deployment_targets[service.type],
         is_public_endpoint: service.is_public_endpoint,
-        created_by: service.created_by,
-        accountID: service.deployment_accounts[0].accountId,
-        region: service.deployment_accounts[0].region
+        created_by: service.created_by
+      }
+      if(service.deployment_accounts){
+        returnObject['accountID'] = service.deployment_accounts[0].accountId,
+        returnObject['region'] = service.deployment_accounts[0].region
+      } else {
+        returnObject['accountID'] = '102707241671',
```

Comment:
We don't want hard-coded account numbers in the code. can u delete this fork and raise another PR?

Reply:
My bad. I am deleting this fork and making a new one and raise a new PR